DashKit/PodStatusExtension.swift

```diff
@@ -0,0 +1,373 @@
+    }
+
+    var reservoir: Double {
+        return Double(reservoirUnitsRemaining) / 100.0
+    }
+}
+
+public extension InternalErrorCode {
+    var description: String {
+        switch self {
+        case .invalidCommand:
+            return "Invalid command"
+
+        case .invalidResponse:
+            return "Invalid response"
+
+        case .incompatibleProductId:
+            return "Incompatible product Id"
+
+        case .unexpectedMessageSequence:
+            return "Unexpected message sequence"
+
+        case .invalidPodId:
+            return "Invalid POD ID"
+        }
+    }
+}
+
+public extension ActivationErrorCode {
+    var description: String {
+        switch self {
+        case .moreThanOnePodAvailable:
+            return "More than one Pod found during Pod activation"
+
+        case .podIsLumpOfCoal1Hour:
+            return "Pod is not activated during 1 hour after UID is set"
+
+        case .podIsLumpOfCoal2Hours:
+            return "Pod is not activated during 2 hour after insulin is filled"
+
+        case .podActivationFailed:
+            return "Other errors during Pod activation"
+
+        case .activationPhase1NotCompleted:
+            return "Try to call activation phase 2 before completing phase 1"
+            
+        case .podIsActivatedOrDeactivating:
+            return "Pod is activated or deactivating"
+        }
+    }
+}
+
+extension PodCommError: LocalizedError {
+    public var errorDescription: String? {
+        switch self {
+        case .unknownError:
+            return "Error not mapped"
+
+        case .phoneNotRegistered:
+            return "Phone is not registered"
+
+        case .podServiceIsBusy:
+            return "Pod service is busy"
+
+        case .podIsNotActive:
+            return "Pod is not active"
+
+        case .failToConnect:
+            return "Failed to connect"
+
+        case .operationTimeout:
+            return "Operation timed out"
+
+        case .notConnected:
+            return "Pod is not connected"
+
+        case .messageSigningFailed:
+            return "Message signinng failed"
+
+        case .podNotAvailable:
+            return NSLocalizedString("Loop cannot locate Pod", comment: "Error text when pod not available.")
+
+        case .bluetoothOff:
+            return "Bluetooth is off"
+
+        case .internalError:
+            return "Internal error"
+
+        case .activationError(let activationError):
+            switch activationError {
+            case .moreThanOnePodAvailable:
+                return NSLocalizedString("More than one Pod discovered", comment: "Error text when multiple pods detected.")
+            default:
+                return "Activation error"
+            }
+
+        case .nackReceived(let nackCode):
+            return "NACK received: \(nackCode)"
+
+        case .podIsInAlarm:
+            return "Pod is in alarm"
+
+        case .systemAlarm:
+            return "System alarm"
+
```

Comment:
Since only some are actually localized, the rest aren't displayed to the user?

Reply:
I wasn't certain initially, but now I think many of them could be displayed, and we don't know exactly which might or might not, so I'll add localization for them all.